.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
```

Comment:
editorconfig doesn't take the place of prettier, it gives a cross-IDE way to configure things like tab configuration. Without that I have to reconfigure my editor not to have it indent things incorrectly. It's a fairly widely used tool -- most editors support a plugin for it.

Reply:
I have no problem with editorconfig per se. I've used it before for projects in other languages. But I don't see what value it adds on top of prettier. Prettier already controls all of the settings specified in this `.editorconfig` (with different values for some of the config). Many editors provide [prettier plugins](https://prettier.io/docs/en/editors.html) that should solve the indentation problems you ran into. It seems preferable to have contributors configure their IDEs to use prettier plugins rather than add additional config.

Reply:
Finally getting back to this a bit; AFAIK prettier only applies things after the fact, whereas editorconfig tells the editor which settings to use for indentation while editing, no? I have prettier plugins installed for vscode and it doesn't seem to configure things in a useful way, but perhaps I'm missing something